README.md

```diff
@@ -18,4 +18,26 @@ If no options or command is specified, `mx` prints information on the available
 
 For an example of `mx` usage, you can read the [Instructions][1] for the Graal project. 
 
+### mx versioning ###
+
+`mx` uses a `major`.`minor`.`patch` versioning scheme.  To figure out if the
+version is sufficient for a given `mx` suite, first compare the major version
+number of your `mx` version against the major number of the required version
+specified in the suite.  If these versions are not equal, you cannot expect
+`mx` to be compatible with this suite.  The minor number has to be greater or
+equal to the specified minor version number.  Compatibility is ensured
+regardless of the patch level.  However, if your patch level is lower than the
+required patch level you might trigger bugs in `mx`.
+
+From a `mx` developer point of view this versioning scheme enforces the following
```

Comment:
Actually, the link you posted says exactly the opposite:
> An is used before words, abbreviations, acronyms, or letters that begin with a **vowel sound**, regardless of their spelling

They even give an example that's probably as close as you can get before "mx" finally gets accepted into dictionaries ;)
>A Microsoft program.
>-but-
>An MS-DOS program.